system/services/validation/EmailCenterValidators.cfc

```diff
@@ -41,23 +48,38 @@ component {
 	public array function existingEmailsUsingInvalidDomains( required string validDomains ) {
 		var delims    = ", #Chr( 10 )##Chr( 13 )#";
 		var domains   = ListToArray( Trim( arguments.validDomains ), delims );
-		var addresses = $getPresideObject( "email_template" ).selectData( distinct=true, selectFields=[ "from_address" ] );
+		var addresses = $getPresideObject( "email_template" ).selectData(
+			  distinct     = true
+			, selectFields = [ "id","from_address" ]
```

Comment:
We should filter here where `from_address` is not null. Also, we should select whether or not this is a system email (for check below).